### PR TITLE
Light2D height range expanded to -2048 to 2048.

### PR DESCRIPTION
**Pull request by ghost**
Fulfills this request: https://github.com/godotengine/godot/issues/27428

### CODE REVIEW

File: scene/2d/light_2d.cpp [hunk 1]
```diff
@@ -438,7 +438,7 @@ void Light2D::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::REAL, "energy", PROPERTY_HINT_RANGE, "0.01,100,0.01"), "set_energy", "get_energy");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "mode", PROPERTY_HINT_ENUM, "Add,Sub,Mix,Mask"), "set_mode", "get_mode");
 	ADD_GROUP("Range", "range_");
-	ADD_PROPERTY(PropertyInfo(Variant::REAL, "range_height", PROPERTY_HINT_RANGE, "-100,100,0.1"), "set_height", "get_height");
+	ADD_PROPERTY(PropertyInfo(Variant::REAL, "range_height", PROPERTY_HINT_RANGE, "-2048,2048,0.1"), "set_height", "get_height");
```
Comment (reduz) [inline scene/2d/light_2d.cpp]: I suggest also adding it like this:
"-2048,2048,0.01,or_greater_or_lesser"
this way you can also specify bigger or smaller numbers that go past these if you want

**Review by reduz — CHANGES_REQUESTED** [review]

**Comment by reduz** [comment]
This is fine, can see why it's needed, check review and it should be fine to merge

File: scene/2d/light_2d.cpp (hunk 1 above)
Comment (avencherus) [reply scene/2d/light_2d.cpp]: 👍 Updated it.

**Review by reduz — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**